### PR TITLE
Junit linesep fixes, getColor fixes, use default font in regression testing

### PR DESCRIPTION
**Pull request by jimandreas**
This pull request addresses issues #4 and #5.   It also changes to loading the default font in the MarkdownBuilderTest.kt - as Travis test engines were not able to load the `inconsolata` font in the android connected tests.

### CODE REVIEW

**Comment by florina-muntenescu** [comment]
Could you rebase, please?

File: TextStyling-Java/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.java [hunk 1]
```diff
@@ -39,9 +40,10 @@
 
     public MarkdownBuilderTest() {
         Context context = InstrumentationRegistry.getTargetContext();
-        int bulletPointColor = context.getColor(R.color.colorAccent);
-        int codeBackgroundColor = context.getColor(R.color.code_background);
-        Typeface codeBlockTypeface = ResourcesCompat.getFont(context, R.font.inconsolata);
+        int bulletPointColor = ContextCompat.getColor(context, R.color.colorAccent);
+        int codeBackgroundColor = ContextCompat.getColor(context, R.color.code_background);
+//        Typeface codeBlockTypeface = ResourcesCompat.getFont(context, R.font.inconsolata);
```
Comment (florina-muntenescu) [inline TextStyling-Java/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.java]: remove the commented line

File: TextStyling-Java/app/src/test/java/com/android/example/text/styling/parser/ParserTest.java [hunk 2]
```diff
@@ -31,45 +31,47 @@
  */
 public class ParserTest {
 
+    private final String LS = System.getProperty("line.separator");
```
Comment (florina-muntenescu) [inline TextStyling-Java/app/src/test/java/com/android/example/text/styling/parser/ParserTest.java]: rename this to `LINE_SEPARATOR`

File: TextStyling-Kotlin/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.kt [hunk 3]
```diff
@@ -34,9 +34,9 @@ import org.junit.Test
 class MarkdownBuilderTest {
 
     private val context = InstrumentationRegistry.getTargetContext()
-    private val bulletPointColor = context.getColor(R.color.colorAccent)
-    private val codeBackgroundColor = context.getColor(R.color.code_background)
-    private val codeBlockTypeface = ResourcesCompat.getFont(context, R.font.inconsolata)
+    val bulletPointColor = ContextCompat.getColor(context, R.color.colorAccent)
```
Comment (florina-muntenescu) [inline TextStyling-Kotlin/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.kt]: Use `getColorCompat` from `Extensions.kt`

File: TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt [hunk 4]
```diff
@@ -23,40 +23,42 @@ import org.junit.Test
  */
 class ParserTest {
 
+    private val separator = System.getProperty("line.separator")
```
Comment (florina-muntenescu) [inline TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt]: make this const and rename to LINE_SEPARATOR

File: TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt [hunk 5]
```diff
@@ -88,28 +90,28 @@ class ParserTest {
     }
 
     @Test fun quoteBulletPointsCode() {
-        val text = "Complex:\n> Quote\nWith points:\n+ bullet `one`\n* bullet `two` is `long`"
+        val text =  """Complex:$separator> Quote${separator}With points:$separator+ bullet `one`$separator* bullet `two` is `long`"""
```
Comment (florina-muntenescu) [inline TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt]: let's decrease the number of quotes here

Comment (jimandreas) [reply TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt]: Done

File: TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt (hunk 4 above)
Comment (jimandreas) [reply TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt]: Refactoring is done, but I am getting complaints from Kotlin about making LINE_SEPARATOR a const :-)

File: TextStyling-Kotlin/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.kt (hunk 3 above)
Comment (jimandreas) [reply TextStyling-Kotlin/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.kt]: Cool - missed that nice extension.   Good stuff.  Done

File: TextStyling-Java/app/src/test/java/com/android/example/text/styling/parser/ParserTest.java (hunk 2 above)
Comment (jimandreas) [reply TextStyling-Java/app/src/test/java/com/android/example/text/styling/parser/ParserTest.java]: Done

File: TextStyling-Java/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.java (hunk 1 above)
Comment (jimandreas) [reply TextStyling-Java/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.java]: Done

File: TextStyling-Java/app/src/test/java/com/android/example/text/styling/parser/ParserTest.java (hunk 2 above)
Comment (jimandreas) [reply TextStyling-Java/app/src/test/java/com/android/example/text/styling/parser/ParserTest.java]: Done

File: TextStyling-Java/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.java (hunk 1 above)
Comment (jimandreas) [reply TextStyling-Java/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.java]: Done

**Comment by jimandreas** [comment]
My pleasure!   Thank you for sharing this awesome java / kotlin project.   I learned a lot!

**Pull request merged**